dev-haskell/hw-kafka-client/hw-kafka-client-5.0.0.ebuild

```
@@ -0,0 +1,35 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.8.2.0
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Kafka bindings for Haskell"
+HOMEPAGE="https://github.com/haskell-works/hw-kafka-client"
+
+LICENSE="MIT"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples"
+
+RDEPEND="dev-haskell/bifunctors:=[profile?]
+	dev-haskell/text:=[profile?]
+	>=dev-lang/ghc-8.8.1:=
+	dev-libs/librdkafka
+"
+DEPEND="${RDEPEND}
+	dev-haskell/c2hs
+	>=dev-haskell/cabal-3.0.0.0
+	test? ( dev-haskell/either
+		dev-haskell/hspec
+		dev-haskell/monad-loops )
+"
+
+src_configure() {
+	haskell-cabal_src_configure \
+		$(cabal_flag examples examples)
```

It's good practice to explicitly enable/disable/`cabal_flag` each available flag:

```suggestion
	haskell-cabal_src_configure \
		$(cabal_flag examples examples) \
		--flag=-it
```

or

```suggestion
	haskell-cabal_src_configure \
		$(cabal_flag examples examples) \
		$(cabal_flag test it)
```